test/libsolidity/smtCheckerTests/invariants/unary_minus_formatting.sol

```diff
@@ -0,0 +1,13 @@
+contract C {
+        int x = -1;
+	function i() public view {
+		assert(x == -1);
+	}
+}
+// ====
+// SMTEngine: chc
+// SMTIgnoreInv: no
+// SMTSolvers: eld
+// ----
+// Info 1391: CHC: 1 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
+// Info 1180: Contract invariant(s) for :C:\n(x = -1)\n
```

Comment:
On this example, the invariant reported previously was `(x = (1))`, now it is `(x = -1)`.